### PR TITLE
Init guide line ios project 1 & 2

### PR DESCRIPTION
**Pull request by tuanlvt-1117**
Guide project ios

### CODE REVIEW

**Review by namtn-2711 — APPROVED** [review]

**Review by tuannm-1276 — APPROVED** [review]

File: IOS/project1.md [hunk 1]
```diff
@@ -1 +1,69 @@
 # Hướng dẫn một số thông tin về IOS Project 1
+
+## 1. Một số thông tin chung
+#### Thời gian thực hiện:
+10 ngày (Tối đa 20 ngày nếu đi part time)
+#### Số lượng thành viên trong team:
+1 ~ 3 người
+#### Chủ đề tham khảo:
+// TODO udpate link late
```
Comment (daolq-2712) [inline IOS/project1.md:9]:
```suggestion
// TODO update link late
```

**Review by vietnh-1109 — APPROVED** [review]

File: IOS/project1.md [hunk 2]
```diff
@@ -1 +1,69 @@
...
+9. MVC
+
+## 2. Các bước thực hiện
+Thực hiện công việc theo từng bước như sau:
+1. Lên ý tưởng, requirements và gửi trainer review theo template
+    https://sal.vn/YZHBxP
+2. Nếu được trainer duyệt thì design prototype trên:
+   https://www.figma.com
+   sau đó gửi trainer review, được duyệt thì chuyển sang bước 3
+3. Breaks tasks như temaplate trên, nhớ sắp xếp theo độ ưu tiên, mỗi task nhỏ hơn 4h
```
Comment (daolq-2712) [inline IOS/project1.md:32]:
```suggestion
3. Breaks tasks như template trên, nhớ sắp xếp theo độ ưu tiên, mỗi task nhỏ hơn 4h
```

Comment (daolq-2712) [inline IOS/project1.md:32]:
```suggestion
3. Breaks tasks như template trên, nhớ sắp xếp theo độ ưu tiên, mỗi task nhỏ hơn 4h
```

File: IOS/project1_en.md [hunk 3]
```diff
@@ -1 +1,69 @@
 # Guideline about IOS Project 1
+
+## 1. General Information
+#### Execution time:
+10 working days
+#### Team size
+1 ~ 3 people
+
+#### Reference topics:
+// TODO udpate link late
```
Comment (daolq-2712) [inline IOS/project1_en.md:10]:
```suggestion
// TODO update link late
```

File: IOS/project2.md [hunk 4]
```diff
@@ -1 +1,17 @@
 # Hướng dẫn một số thông tin về IOS Project 2
+
+## Các thông tin và bước thực hiện project 2 tương tự như project 1 ngoài có một số điểm khác sau đây
+#### Thời gian thực hiện:
+8 ngày (Tối đa 16 ngày nếu đi part time)
+
+#### Danh sách project có thể tham khảo
+// TODO udpate link late
```
Comment (daolq-2712) [inline IOS/project2.md:8]:
```suggestion
// TODO update link late
```

File: IOS/project2_en.md [hunk 5]
```diff
@@ -1 +1,17 @@
 # Guideline about IOS Project 2
+
+## The information and steps to implement project 2 are similar to project 1 apart from having some other points as follows:
+#### Execution time:
+8 working days 
+
+#### List of projects you can refer to:
+// TODO udpate link late
```
Comment (daolq-2712) [inline IOS/project2_en.md:8]:
```suggestion
// TODO update link late
```

**Review by daolq-2712 — CHANGES_REQUESTED** [review]
Pls check my comments!

**Comment by daolq-2712** [comment]
[LGTM](https://i.imgur.com/jsFLAVe.gif)

**Review by daolq-2712 — APPROVED** [review]
LGTM

**Pull request merged**